src/Alarm.cpp

```diff
@@ -127,11 +128,11 @@ void Alarm::disarm(){
   _alarmLED->off();
   _isSilenced = false;
   _laser->off();
-  Logger::Log("Alarm has been disarmed");
+  Serial.write(34002); //34002 = Command, Info, Execute Command: Disarm
```

Comment:
`002` isn't really appropriate here. Thats an execute command. It needs to have its own logging statement. It should say something about 'successfully disarmed' or something.